openzwavemqtt/models/value.py

```diff
@@ -95,6 +95,20 @@ def write_only(self) -> bool:
         """Return WriteOnly."""
         return self.data.get("WriteOnly")
 
+    @property
+    def value_list(self) -> list:
+        """Return List in Values."""
+        if not isinstance(self.value, str) and isinstance(self.value, dict):
+            return list(map(lambda x: x["Label"], self.value.get("List")))
+        return None
+
+    @property
+    def value_selected(self) -> str:
+        """Return Selected Value."""
+        if "Selected" in self.value:
+            return self.value.get("Selected")
```

Comment:
Shouldn't this be Selected_id ?

Reply:
I certainly understand that the labels may be dynamic. The use case for this change has to do with the new climate component I submitted a PR for [here](https://github.com/cgarwood/homeassistant-zwave_mqtt/pull/101). The reason I had chosen to use labels instead of the id was to mimic the (presumed) function of ``data_items`` from the previous homeassistant zwave component. The climate module uses the labels to determine which HVAC_MODE_* to use or if to use a preset. 

Obviously I can move any label retrieval to the climate module, and it seems that may be what I need to do. If I were to do that, I agree that the actual benefit of having the helper method/property is decreased. 

I suppose I'm thinking that there will be plenty of people in the future writing code to obtain only the labels and then converting labels to values for ``send_value``.  These helper functions would make that unnecessary. Additionally. the current formatting of the Value (I installed OZW daemon yesterday) seems less than helpful, returning ``[{"Value":0, "Label": "Off"}]`` where a simple dict of ``{"0":"Off", "1":"On"}`` seems like it would be simpler to use as it can be referred to as simply key,value.

Reply:
The only thing the label can be used for is displaying to the user in the frontend. Not for actually setting values, comparing or whatsoever. Those labels can be changed in the OZW configfiles. Better to only look at the ID's and map those ID's to the Home Assistant values.

The only repeating thing that can be improved is getting the label for a value.
so a method like get_value_label(value_id: int): --> str

Are you comfortable with closing this PR and handle the values listing in your climate PR ?
We can (later) add the helper method to the library for the get_value_label if we still see a usecase.

Reply:
Yeah, absolutely, I'll close it. I'll take a look at everything over the next few days. I can certainly see how a ``get_label_value`` could be useful.